### PR TITLE
[13.0][MIG] account_payment_return: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Main changes:
- Adapt python code
- Adapt tests
  - I removed test_find_match_move_duplicate method in tests because now in v13 it won't happen that there are two account.move with the same name, there is a constraint in account.move called _check_unique_sequence_number that restricts this
- Adapt views
- Extend ShowPaymentInfo template instead of completely replacing it

Cc @Tecnativa TT19691

### CODE REVIEW

File: account_payment_return/models/account_journal.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    default_expense_account_id = fields.Many2one(
+        comodel_name="account.account",
+        string="Default Charges Account",
+        help="Default account for bank charges",
+    )
+    default_expense_partner_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Default Charges Partner",
+        domain=[("supplier", "=", True)],
+        help="Default partner for " "charge expenses",
```
Comment (kittiu) [inline account_payment_return/models/account_journal.py]: Remove space.

File: account_payment_return/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
+* 7 i TRIA <http://www.7itria.cat>
+* Avanzosc <http://www.avanzosc.com>
+* Pedro M. Baeza <pedro.baeza@tecnativa.com>
+* Markus Schneider <markus.schneider@initos.com>
+* Sergio Teruel <sergio.teruel@tecnativa.com>
+* Carlos Dauden <carlos.dauden@tecnativa.com>
+* David Vidal <david.vidal@tecnativa.com>
+* Luis M. Ontalba <luis.martinez@tecnativa.com>
```
Comment (kittiu) [inline account_payment_return/readme/CONTRIBUTORS.rst]: You can add yourself as contributor here if you wish.

File: account_payment_return/models/account_journal.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    )
+    return_auto_reconcile = fields.Boolean(
+        string="Reconcile payment returns",
+        help="Enable automatic payment return reconciliation. This option "
+        "is meant to be used only when working when transfer accounts, "
```
Comment (kittiu) [inline account_payment_return/models/account_journal.py]:
```suggestion
        "is meant to be used only when working with transfer accounts, "
```

File: account_payment_return/models/account_move.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        for returned_reconcile in returned_reconciles:
+            payment = returned_reconcile.credit_move_id
+            payment_ret = returned_reconcile.debit_move_id
+            values.append(
+                {
+                    "name": payment.name,
+                    "journal_name": payment.journal_id.name,
+                    "amount": returned_reconcile.amount,
+                    "currency": self.currency_id.symbol,
+                    "digits": [69, self.currency_id.decimal_places],
```
Comment (kittiu) [inline account_payment_return/models/account_move.py:46]: Sorry, just wondering what number 69 represent?
Saw this also on odoo core, but also don't know what it is.

File: account_payment_return/models/payment_return.py [hunk 5]
```diff
@@ -0,0 +1,477 @@
...
+
+
+class PaymentReturn(models.Model):
+    _name = "payment.return"
+    _inherit = ["mail.thread"]
+    _description = "Payment return"
+    _order = "date DESC, id DESC"
+
+    company_id = fields.Many2one(
+        "res.company",
```
Comment (kittiu) [inline account_payment_return/models/payment_return.py]:
```suggestion
        comodel_name="res.company",
```

**Review by kittiu — APPROVED** [review]
Functional tested, LGTM

**Comment by ernestotejeda** [comment]
@kittiu , changes done

**Comment by ernestotejeda** [comment]
I also updated **[IMP] account_payment_return: black, isort** commit with changes made by prettier xml

File: account_payment_return/models/account_move.py [hunk 6]
```diff
@@ -1,19 +1,73 @@
...
+                    "name": payment_ret.name,
+                    "journal_name": payment_ret.journal_id.name,
+                    "amount": -returned_reconcile.amount,
+                    "currency": self.currency_id.symbol,
+                    "digits": [69, self.currency_id.decimal_places],
+                    "position": self.currency_id.position,
+                    "date": payment_ret.date,
+                    "payment_id": payment_ret.id,
+                    "move_id": payment_ret.move_id.id,
+                    "ref": payment_ret.move_id.name,
```
Comment (carlosdauden) [inline account_payment_return/models/account_move.py]:
```suggestion
                    "ref": '{} ({})'.format(payment_ret.move_id.name, payment_ret.move_id.ref),
```

**Review by carlosdauden — APPROVED** [review]
Minor suggestion

**Comment by ernestotejeda** [comment]
Changes done @carlosdauden

**Pull request merged**